service-commands/scripts/seed.js

```diff
@@ -0,0 +1,194 @@
+const { program } = require('commander')
+const ServiceCommands = require('../src/index')
+const _ = require('lodash')
+const colors = require('colors')
+
+const {
+    SeedSession,
+    SeedUtils,
+    Constants
+} = ServiceCommands
+
+const {
+  parseMetadataIntoObject,
+  camelToKebabCase,
+  parseSeedActionRepeatCount,
+  passThroughUserInput
+} = SeedUtils
+
+const {
+  SERVICE_COMMANDS_PATH
+} = Constants
+
+// See below file for entrypoint to adding new libs API calls to CLI
+const CLI_TO_COMMAND_MAP = require('../src/commands/seed/cliToCommandMap')
+
+// Setting pretty print colors
+colors.setTheme({
+  happy: 'rainbow',
+  success: 'green',
+  info: 'magenta',
+  error: 'red'
+})
+
+const checkExecutedFromCorrectDirectory = () => {
+  if (process.cwd() !== SERVICE_COMMANDS_PATH) {
+    throw new Error(
+      '`A seed` must be run from the service-commands directory. This is because of the way relative paths for node-localstorage work.'.error
+    )
+  }
+}
+
+program
+  .command('create-user')
```

Comment:
Create user and user management are the primary parts of this that require explicit declaration via `program.command` and a bit more wrapper logic to account for how we manage local state via the seed cache JSON and hedgehog entropy key in localstorage.